File: c/01-estrutura-sequencial/consumo.c
/*Problema "consumo"
Fazer um programa para ler a distância total (em km) percorrida por um carro, bem como o total de
combustível gasto por este carro ao percorrer tal distância. Seu programa deve mostrar o consumo
médio do carro, com três casas decimais.*/

#include <stdio.h> 
int main () {
    double averagefuel, distance, spentfuel; 
    printf("Distancia percorrida: ") ;
    scanf("%lf", &distance);
    printf("Combustivel gasto: "); 
    scanf("%lf", &spentfuel);
    
    averagefuel = distance / spentfuel;
    printf("Consumo medio: %.3lf\n",averagefuel);
   
    return 0;
}

File: java/01-estrutura-sequencial/Consumo.java
/*Problema "consumo"
Fazer um programa para ler a distância total (em km) percorrida por um carro, bem como o total de
combustível gasto por este carro ao percorrer tal distância. Seu programa deve mostrar o consumo
médio do carro, com três casas decimais.*/

import java.util.Locale;
import java.util.Scanner;

public class Consumo {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double distance, spentfuel, average_fuel;
		
		System.out.print("Distancia percorrida: ");
		distance = sc.nextDouble();
		System.out.print("Combustivel gasto: ");
		spentfuel = sc.nextDouble();
		
		average_fuel = distance / spentfuel;
		
		System.out.println("Consumo medio: " + String.format("%.3f", average_fuel));
		
		sc.close();
	}

}
